pkg/controller/remotemachineset/awsactuator.go

```diff
@@ -43,6 +44,8 @@ var (
 
 	// reg is a regex used to fetch condition message from error when subnets specified in the MachinePool are invalid
 	reg = regexp.MustCompile(`^InvalidSubnetID\.NotFound:\s+([^\t]+)\t`)
+
+	versionsSupportingSpotInstances = semver.MustParseRange(">=4.5.0")
```

Comment:
`MustParseRange` panics when range cannot be parsed... Would it be better to use `ParseRange` instead? We can also just log the error and move on without blocking the flow

Reply:
This is done in the init of the package. If the range cannot be parsed, that is a coding error. We want the program to crash in this case.